controls/nist_ocp4.yml

```diff
@@ -15888,17 +15888,40 @@ controls:
   - high
   - moderate
 - id: SI-7(2)
-  status: pending
+  status: automated
   notes: |-
+    The OpenShift Platform provides the File Integrity Operator to do
+    integrity verification of files and information on the nodes. [1]
+
+    The operator can be configured to perform integrity checks at user-defined
+    intervals. [2]
+
+    A Prometheus rule will be added for the File Integrity Operator to alert
+    organization defined personnel for any integrity check failure. 
+
+    [1] https://docs.openshift.com/container-platform/latest/security/file_integrity_operator/file-integrity-operator-installation.html
+    [2] https://docs.openshift.com/container-platform/4.6/security/file_integrity_operator/file-integrity-operator-configuring.html
+
     A control response for SI-7(2) is planned. Engineering progress can be
     tracked via:
 
     https://issues.redhat.com/browse/CMP-545
-  rules: []
+
+  rules:
+  - file_integrity_exists
+  - file_integrity_notification_enabled
+
   description: |-
-    The organization employs automated tools that provide notification to [Assignment: organization- defined personnel or roles] upon discovering discrepancies during integrity verification.
+    The organization employs automated tools that provide notification
```

Comment:
Is this just reformatting? (`fmt -78` or such?)

Reply:
Yes, it was just reformatted